test/e2e/app_management_test.go

```diff
@@ -1747,6 +1747,32 @@ func TestSyncOptionPruneFalse(t *testing.T) {
 		Expect(ResourceSyncStatusIs("Pod", "pod-1", SyncStatusCodeOutOfSync))
 }
 
+// make sure that if we deleted a resource from the app, it is not pruned if the app has the sync option Prune=false
+func TestSyncOptionSyncOptionPruneFalse(t *testing.T) {
```

Comment:
add e2e test when option is specified at the application level and resource level